vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -859,6 +859,9 @@ class TimelineFragment @Inject constructor(
         views.locationLiveStatusIndicator.stopButton.debouncedClicks {
             timelineViewModel.handle(RoomDetailAction.StopLiveLocationSharing)
         }
+        views.locationLiveStatusIndicator.root.debouncedClicks {
```

Comment:
Not tested, but is it mandatory to use `root`?

I was wondering if `views.locationLiveStatusIndicator.debouncedClicks {` would work the same way.

Reply:
Yes indeed, the `root` is not needed, I removed it.